src/features/components/organisms/greyed-container/trade-benefit.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { item_container } from './style.module.scss'
+import { TradeBenefitType } from './types'
+import Flex from 'features/components/atoms/flex-box'
+import { Localize } from 'components/localization'
+import Typography from 'features/components/atoms/typography'
+import Image from 'features/components/atoms/image'
+
+interface TradeBenefitProps {
+    item: TradeBenefitType
+}
+
+const TradeBenefit = ({ item }: TradeBenefitProps) => {
+    return (
+        <Flex.Box className={item_container} direction="col" align="center" gap="5x">
+            <Image src={item.icon_src} width={48} height={48} />
+
+            <Flex.Item>
+                <Typography.Paragraph size="medium" textcolor="inverted" align="center">
```

Comment:
`size="medium"` by default
```suggestion
                <Typography.Paragraph textcolor="inverted" align="center">
```